addons/sale/wizard/sale_order_cancel_views.xml

```diff
@@ -1,18 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="sale_order_cancel_view_form" model="ir.ui.view">
-        <field name="name">sale.order.cancel.form</field>
+        <field name="name">sale.sale.order.cancel.form</field>
```

Comment:
No need to modify this line, the current logic model_name + view_type is clear enough, no need to add the module ;)